src/lib.rs

```diff
@@ -45,6 +45,10 @@ pub use crate::Either::{Left, Right};
 /// preference.
 /// (For representing success or error, use the regular `Result` enum instead.)
 #[cfg_attr(feature = "serde", derive(serde::Serialize, serde::Deserialize))]
+#[cfg_attr(
+    feature = "borsh",
+    derive(borsh::BorshSerialize, borsh::BorshDeserialize)
+)] // encdoed as u8 discriminant prefix + value
```

Comment:
Typo:

```suggestion
)] // encoded as u8 discriminant prefix + value
```

I looked at `cargo expand` though, and I don't really like that it also adds a public `impl borsh::de::EnumExt for Either`. Is there any reason to *want* that? How hard would it be to not use the derive?